src/ui/layout/BrowseBar.vue

```diff
@@ -36,7 +36,7 @@
           ref="objectName"
```

Comment:
i think you should look to add a new attribute like "lockedBy" so that we can see which items were locked by this utility instead of general users

Reply:
Done